slackbot/plugins/update.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+if [[ $ == 0 ]]; then
+for d in */; do
+    echo $d
+    cd $d
+    git fetch
+    git pull
+    cd ..
+done
+else
+    for d in "$@"; do
+      echo $d
+      cd $d
+      git fetch
+      git pull
+      cd ..
+    done
+fi
```

Comment:
I think knowing the function of updating is something that would be valuable to anyone setting up the bot. we can just put it in the git ignore